fms/models/llama.py

```diff
@@ -53,6 +53,8 @@ class LLaMAConfig(ModelConfig):
     mlp_bias: bool = False
     tie_heads: bool = False
     rope_theta: float = 10_000.0
+    linear_config: Optional[Mapping[str, Any]] = None
```

Comment:
We should test that this works properly with serialization and deserialization of the config